docs/source/index.rst

```diff
@@ -28,13 +29,16 @@ Resources
 * `GitHub Repo <https://github.com/airbnb/binaryalert>`_
 * `Blog Post <https://medium.com/airbnb-engineering/binaryalert-real-time-serverless-malware-detection-ca44370c1b90>`_
 * `Slack <https://binaryalert.herokuapp.com/>`_
-* `Twitter <https://twitter.com/binaryalert_io>`_ (unofficial)
+* `Twitter <https://twitter.com/binaryalert_io>`_
```

Comment:
re-add back unofficial (its for legal/pr purposes)

Reply:
Ah ok. Can I (in the troubleshooting section) say that we can be contacted via Slack?

Reply:
Yup that's fine, so long as it's labeled as unofficial above